src/features/auth/LoginForm.tsx

```diff
@@ -49,6 +49,7 @@ export const LoginForm = ({
             label={t('auth:data.username.label')}
             required={t('auth:data.username.required')}
             formatValue={(v) => v?.toLowerCase().trim()}
+            autoFocus
```

Comment:
Be careful, in the issue it is written "not on a mobile device" 😉  
Chakra UI provides you ways to know on which device you are running the app using `useBreakpointValue` hook. There is an example of this hook in `DateSelectorPicker` component